keylime/tenant.py

```diff
@@ -205,6 +205,8 @@ def process_allowlist(self, args):
             # Process allowlists
             al_data["excllist"] = excl_data
             self.allowlist = al_data
+        elif excl_data is not None:
+            raise UserError("--allowlist must be used alongside --exclude")
```

Comment:
Maybe a slightly better wording is something like `--exclude cannot be used without an --allowlist`.

Reply:
Done!